File: ClasesVB-C#/Simulacro_parcial_1_2018/simulacro_parcial/C#_RegistroUsuarios/RegistroUsuarios/BDHelper.cs
using System;
using System.Collections.Generic;
using System.Data;
using System.Data.SqlClient;
public class BDHelper
{
    private string string_conexion = "Data Source=maquis;Initial Catalog=DB_Usuarios;User id=avisuales;Password=avisuales;";

    private static BDHelper instance; //Unica instancia de la clase
    public static BDHelper getBDHelper()
    {
        if (instance == null)
            instance = new BDHelper();
        return instance;
    }

}

File: ClasesVB-C#/Simulacro_parcial_1_2018/simulacro_parcial/VB_RegistroUsuarios/RegistroUsuarios/BDHelper.vb

Imports System.Data.SqlClient

Public Class BDHelper
    Private string_conexion As String = "Data Source=maquis;Initial Catalog=DB_Usuarios;User id=avisuales;Password=avisuales;"

    Private Shared instance As BDHelper 'Unica instancia de la clase

    Public Shared Function getBDHelper() As BDHelper
        If IsNothing(instance) Then
            instance = New BDHelper()
        End If
        Return instance
    End Function

End Class
